src/public/css/base.css

```diff
@@ -1,8 +1,12 @@
+html {
+  background-color: var(--background);
```

Comment:
so background was already on body and wasn't doing the trick. This one seems to correctly change the background to light-black when in dark theme.